src/layout/header/index.js

```diff
@@ -5,20 +5,15 @@ import { withRouter } from "react-router";
 import ArrowBackIcon from "@mui/icons-material/ArrowBack";
 import "./header.css";
 
-const Header = (props) => {
-  let backIcon;
-  if (props.location.pathname !== "/") {
-    backIcon = (
-      <ArrowBackIcon
-        className="go-back"
-        onClick={() => props.history.goBack()}
-      />
-    );
-  }
+const Header = ({ history, location }) => {
   return (
     <AppBar className="app-header">
       <Toolbar>
-        {backIcon}
+        {location.pathname !== "/" ? (

```

Comment:
No need for short `if` here, you can just do {location.pathname !== "/" && <ArrowBackIcon className="go-back" onClick={() => history.goBack()} />}

Reply:
Will dot it. Thank you !